gateway_to_backend/protocol_buffers_files/error.proto

```diff
@@ -4,6 +4,12 @@ package wirepas.proto.gateway_api;
 
 enum ErrorCode {
     option allow_alias = true;
+    // First enum code is the one returned in case error code
+    // is unknown on the other side (new added values in future)
+    // To avoid having OK as default code, better to use an explicit UNKNOWN_ERROR_CODE.
+    // It will become the default only for the one using a protobuf file that has this
+    // error code defined.
+    UNKNOWN_ERROR_CODE = 99;
```

Comment:
Does proto2 allow negative enum values, so -1 would be more clearer?

Reply:
Yes it seems to be supported. It is inefficient on the wire but as this value is never supposed to go on wire it is fine to use -1.

I'll change it to -1, good feedback!